apps/qa/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@fullstory/browser": "^1.7.1",
+    "@babel/traverse": ">=7.23.2",
```

Comment:
Do we need this dependency? I don't see it being used anywhere and when I removed it everything seemed to work as expected.